torchgeo/datasets/potsdam.py

```diff
@@ -0,0 +1,400 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""Potsdam dataset."""
+
+import os
+from typing import Any, Callable, Dict, Optional
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pytorch_lightning as pl
+import rasterio
+import torch
+from matplotlib.figure import Figure
+from PIL import Image
+from torch import Tensor
+from torch.utils.data import DataLoader
+from torchvision.transforms import Compose
+
+from ..datasets.utils import dataset_split, draw_semantic_segmentation_masks
+from .geo import VisionDataset
+from .utils import check_integrity, extract_archive, rgb_to_mask
+
+
+class Potsdam2D(VisionDataset):
+    """Potsdam 2D Semantic Segmentation dataset.
+
+    The `Potsdam <https://www2.isprs.org/commissions/comm2/wg4/benchmark/2d-sem-label-potsdam/>`_
+    dataset is a dataset urban semantic segmentation used in the 2D Semantic Labeling
+    Content - Potsdam. This dataset uses the "4_Ortho_RGBIR.zip" and "5_Labels_all.zip"
```

Comment:
This sentence is unclear to me, is Content supposed to be Contest? What does the "- Potsdam" at the end mean?

Reply:
That's a typo. It should be Contest.

Reply:
The contest website splits the 2D Semantic Labeling Contest into 2 sections: Potsdam and Vaihingen. See https://www2.isprs.org/commissions/comm2/wg4/benchmark/2d-sem-label-potsdam/